src/drive/ducks/upload/styles.styl

```diff
@@ -158,16 +158,19 @@ progress.upload-queue-progress::-moz-progress-bar
     $coz-bar-border-size = 1px
 
     .upload-queue
-        position static
+        top $coz-bar-size
```

Comment:
There's no position anymore ?

Reply:
there is but it keeps the fixed position set for desktop
https://github.com/GoOz/cozy-drive/blob/603682218a6fc3d18963c5bc406fe562b4283093/src/drive/ducks/upload/styles.styl#L6